compiler/rustc_target/src/spec/armv7_unknown_linux_uclibceabihf.rs

```diff
@@ -0,0 +1,23 @@
+use crate::spec::{Target, TargetOptions};
```

Comment:
Please leave a comment in this file with your preferred way of being contacted when someone touches this target.

Reply:
I think this documentation should go into https://github.com/rust-lang/rust/tree/master/src/doc/rustc/src/platform-support, not target definition files -- I think we should also work on a template with some other things that new targets should fill out (similar to the existing file in there).

Notably, I think this target doesn't yet have the following requirement fulfilled, though it may be something that we document for *all* uclibc targets in a common place and then link to (still needs to happen though).

> The target must provide documentation for the Rust community explaining how to build for the target, using cross-compilation if possible. If the target supports running tests (even if they do not pass), the documentation must explain how to run tests for the target, using emulation if possible or dedicated hardware if necessary.

Reply:
Thanks for the feedback!  I'm glad to do so.  Is there a high-quality example of such instructions for another platfrm which you could point me towards?  That would give me something to imitate.